### PR TITLE
Wagtail 5.1: Add tests in main.yml, Update docs

### PR DESCRIPTION
**Pull request by katdom13**
Upstream PR: https://github.com/Frojd/wagtail-geo-widget/pull/121/

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -7,11 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
         wagtail-version:
           - 4.1.5
           - 4.2.3
```
Comment (nickmoreton) [inline .github/workflows/main.yml]: Could these be changed to
```suggestion
          - 4.1
          - 4.2
```

**Review by nickmoreton — CHANGES_REQUESTED** [review]
Hi @katdom13 I left a suggestion. which should then test on latest of both 4.1 and 4.2 versions. Current latest is 4.1.6 4.2.4

File: .github/workflows/main.yml (hunk 1 above)
Comment (katdom13) [reply .github/workflows/main.yml]: Removed patches:
https://github.com/torchbox-forks/wagtail-geo-widget/pull/1/commits/77d8821994d115628830f65c524924eb647f140d

**Review by nickmoreton — APPROVED** [review]
Thanks @katdom13 I approve.

**Pull request merged**